testdata/releases.yaml

```diff
@@ -17,7 +17,7 @@
     status: deployed
   chart:
     metadata:
-      name: prothos-chart
+      name: porthos-chart
```

Comment:
Three Musketeers is a historical adventure novel written in 1844 by French author Alexandre Dumas ... D'Artagnan offers the letter to Athos, Porthos, and Aramis